gradle/libs.versions.toml

```
@@ -18,6 +18,8 @@ kotlinx-io = "org.jetbrains.kotlinx:kotlinx-io-core:0.6.0"
 compose-collection = { module = "org.jetbrains.compose.collection-internal:collection", version.ref = "compose" }
 compose-runtime = { module = "org.jetbrains.compose.runtime:runtime", version.ref = "compose" }
 
+androidx-annotation = { module = "androidx.annotation:annotation", version = "1.9.1" }
```

I prefer the short form if we are not sharing the version
```suggestion
androidx-annotation = "androidx.annotation:annotation:1.9.1"
```